### PR TITLE
Update to use externpro 21.04.1

### PR DESCRIPTION
**Pull request by brobeson**

### CODE REVIEW

**Comment by brobeson** [comment]
@bnhhughes @swflb I need to merge and tag this so I can move on with building projects that require ovsrpro. If you have any concerns, add comments and I can address them in a new pull request.

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos/devtoolset-6-toolchain-centos7:latest
+FROM centos/devtoolset-7-toolchain-centos7:latest
```
Comment (swflb) [inline .devcontainer/Dockerfile:1]: is devtoolset-9 not available?

**Review by swflb — COMMENTED** [review]
#Approved

**Comment by bnhhughes** [comment]
Sorry for the slow response - looks good, #approved

**Pull request merged**